src/registrar/utility/csv_export.py

```diff
@@ -1164,6 +1198,29 @@ class DomainDataFederal(DomainExport):
     Inherits from BaseExport -> DomainExport
     """
 
+    # NOTE - this override is temporary. Delete this after we consolidate these @property fields.
```

Comment:
What do you mean by consolidate the @property fields? Can you change this to describe when this section can be removed?